### DISCUSSION TITLE
Welcome to the Notabase community!

### DISCUSSION THREAD

**Discussion by churichard** [churichard/notabase#1] [General]
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We hope that you:
* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you're planning to do with Notabase!

**Comment by TaylanTatli** [comment]
Hi! I fall in love with Notabase. But I couldn't find any export/sync option and because it's in alpha phase, I'm afraid to lost all my notes. Also it's hard to use in mobile, clicking links etc.

Forgot to mention: also can't hide sidebar on mobile.

I will definitely pay for it when it becomes more mature and if it doesn't cost too much. (1$ = 8,62₺ right now 🤦🏻‍♂️)

**Reply by churichard** [reply]
Hi Taylan! Thank you for the feedback, I really appreciate it.

Being able to export notes into markdown format is a priority for me, and I'll be adding it soon. The database is also backed up automatically every day, so if the notes happen to be lost somehow, they can always be restored to the most recent backup. I definitely don't want people to feel like they're locked into the app or that they might lose their notes. It's your data and you should have full control and access to it.

I've been focused primarily on the desktop experience during the alpha, but better mobile support is also important and is on the roadmap.

If you have any other feedback or ideas, feel free to let me know! It's very helpful for me.

**Reply by churichard** [reply]
Hey Taylan, just want to follow up here. I added the ability to export your notes to markdown, and you can now hide the sidebar. I'll be making more improvements for better mobile support in the near future. Thanks for your support!

**Reply by TaylanTatli** [reply]
> Hey Taylan, just want to follow up here. I added the ability to export your notes to markdown, and you can now hide the sidebar. I'll be making more improvements for better mobile support in the near future. Thanks for your support!

Hi! Sorry for late reply. :disappointed:  Yep, I saw it, and it's much better on mobile now. About exporting, it seems we can only export per post, not all of them.

**Reply by churichard** [reply]
No worries. I just added the ability to export all notes. Let me know if you run into any other issues!

**Reply by TaylanTatli** [reply]
Wow, you are really fast! :open_mouth:
